### DISCUSSION TITLE
Maintainability Question

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [Arduino-IRremote/Arduino-IRremote#987] [Ideas]
I was wondering if you maintainers are of the open minded
approach that let's change affect repositories for the better.

<br>

For one, what is the current method of providing docs?
- The README
- The wiki
- The github pages page

(all three are present, which begs the question why that is)

<br>

The wiki has been linked as the `old wiki` so I assume it
definitely isn't that option, though why is it still here, is
there still content that hasn't been transferred to the 
newer documentation?

<br>

The github pages page seems to only be used for API
references if I'm correct, which would make the README
the primary documentation?

<br>

---

<br>

##### If this is the case my typical action plan would be:
- Separate README sections into topics -> `Documentation/Topic.md`
- Make direct links into link references
- Make topics reachable in README

<br>

##### And for the wiki:
- Check every wiki page for relevant content
- Move contents from wiki page to `Documentation/Topic.md 
- Disable wiki

<br>

##### On the topic of disabling:
I'd like to suggest disabling unused features,
such as in this case the `Packages` widget.

**Comment by ArminJo** [comment]
> I'd like to suggest unused features, such as in this case the Packages widget.

Do you know how to disable?
Thanks

**Reply by ElectronicsArchiver** [reply]
Certainly:
- Switch to the main repository view
- Click the `cog` icon in the upper right corner
- Uncheck the `packages` option in the lower list

Thanks

**Reply by ArminJo** [reply]
👍 🥇 
Thanks!!!

**Reply by ArminJo** [reply]
Thanks again!
I now removed the old wiki, all was contained in Ken Shirriffs blog, and a link to the blog was added.

I personally have problems with separating the readme into different files/topics, since then easy searching a keyword is no longer possible.
But your idea was very good and so I included a TOC instead 😀.

For me this is all I can do, If you still have some suggestions, let me know.

Regards
Armin

**Reply by ElectronicsArchiver** [reply]
> I personally have problems with separating the readme into different files/topics, since then easy searching a keyword is no longer possible.

Well you could use a GitHub Pages site with JustTheDocs like seen here
https://electronicsarchiver.github.io/Astro8-Computer/

So in general do you mind me formatting the 
README as long as I'm not splitting it up?

**Reply by ArminJo** [reply]
Please do it. I cannot promise to accept all changes, but I am sure there are many ideas I will incorporate.

**Reply by ElectronicsArchiver** [reply]
@ArminJo Btw, do you have anything against a generated README?

Even if you don't want to have the README itself in parts,
just for editing / maintaining, having it split up would
make sense.

I could add a simple workflow that combines section into the README~

**Reply by ArminJo** [reply]
Thanks but I am fine with one file 😀.
